### ISSUE TITLE
When eating too much, "Full" effect is displayed twice

### ISSUE THREAD

**Issue by CoroNaut** [CleverRaven/Cataclysm-DDA#70078]
### Describe the bug

The "Full" effect on speed is displayed twice, but I think it still only counts it as one debuff.
"Happy" also seems to be displayed twice.
"Engorged" also twice
### Attach save file

[testinggg-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/13614367/testinggg-trimmed.tar.gz)

### Steps to reproduce

1. Eat enough to be "Full"
2. Effect is displayed twice

### Expected behavior

"Full" displays only once

### Screenshots

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/24307650/d556c27f-83f2-48fd-b673-d139ffdfaaa3)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/24307650/329f0b46-b81b-4d9c-9011-976ad00afb29)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/24307650/e31ca38e-560a-407f-b748-76e42560dd28)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/24307650/6e4e0a6f-574c-46e8-850f-2775383815f4)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/24307650/ca29e186-0338-44e2-bab4-fc8004d55c39)

### Versions and configuration

- OS: Windows
    - OS Version: MINGW/CYGWIN/MSYS2 on unknown Windows version
- Game Version: 7773670 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Slowdown Fungal Growth [no_fungal_growth]
]

**Comment by RenechCDDA** [comment]
Duplicate of #68633
